src/test/scala/org/constellation/TransactionHashIntegrityTest.scala

```diff
@@ -1,26 +1,17 @@
 package org.constellation
 
 import cats.effect.{ContextShift, IO}
-import cats.implicits._
-import org.constellation.keytool.{KeyStoreUtils, KeyUtils}
+import org.constellation.keytool.KeyStoreUtils
 import org.constellation.primitives.Transaction
 import org.constellation.serializer.KryoSerializer
 import org.constellation.wallet.{KryoSerializer => WalletKryoSerializer}
 import org.constellation.wallet.{Hashable, Transaction => WalletTransaction}
 import org.scalatest.{FreeSpec, Matchers}
 import io.circe.generic.auto._
 import io.circe.parser.parse
-import org.constellation.schema.Id
-import org.http4s.{Header, Headers, Request, Uri}
-import org.http4s.client.dsl.io._
-import org.http4s.client.dsl._
-import org.http4s.Method._
-import pl.abankowski.httpsigner.http4s.Http4sRequestVerifier
-import pl.abankowski.httpsigner.signature.generic.GenericVerifier
```

Comment:
:+1: sorry, I've been testing sth and forgotten about remove

Reply:
Nbd